homeassistant/components/media_player/liveboxplaytv.py

```diff
@@ -92,7 +92,7 @@ def update(self):
                     self._client.get_current_channel_image(img_size=300)
                 self.refresh_channel_list()
         except requests.ConnectionError:
-            self._state = STATE_OFF
+            self._state = STATE_UNKNOWN
```

Comment:
Default to `None`.

Reply:
Why?

Reply:
The `Entity` base class will handle converting `None` to `STATE_UNKNOWN`.

Reply:
Uh okay. I'll edit that.

Reply:
Should the state also be set to `None` in the `__init__` then?

Reply:
Yes! Good!